SpatialGDK/Source/SpatialGDK/Private/SpatialView/WorkerView.cpp

```diff
@@ -41,6 +41,8 @@ void WorkerView::SendAddComponent(Worker_EntityId EntityId, ComponentData Data,
 	EntityViewElement* Element = View.Find(EntityId);
 	if (ensure(Element != nullptr))
 	{
+		checkf(!Element->Components.ContainsByPredicate(ComponentIdEquality{ Data.GetComponentId() }),
```

Comment:
Should this be check slow?

Reply:
I'd vote against - this is very important, and DebugGame is used very sparsely.